### ISSUE TITLE
Uncaught TypeError: viewport.addEventListener is not a function...

### ISSUE THREAD

**Issue by micheleivani** [xtianmiller/emergence.js#3]
Hi, I'm playing your package. It seams that it sweats my needs. Good job. :) Back to the issue. I'm using Laravel Mix for handling all js and css assets.

In my main js file `app.js` I have just added these simple lines:

```
import emergence from 'emergence.js';
emergence.init();
```

After the Mix compilation `npm run dev` I included the final js file in my html. Here is the error I get:

```
app.js:292 Uncaught TypeError: viewport.addEventListener is not a function
    at Object.emergence.init (app.js:292)
    at Object.<anonymous> (app.js:79)
    at __webpack_require__ (app.js:20)
    at Object.<anonymous> (app.js:364)
    at __webpack_require__ (app.js:20)
    at Object.defineProperty.value (app.js:66)
    at app.js:69

```

Hope you can help me.

**Comment by winstef** [comment]
Same issue for me

**Comment by xtianmiller** [comment]
Thanks for the feedback. I suspect it may be caused by your final js (I assume) being loaded in the head, and therefore emergence gets initiated before the dom content is loaded. Placing the js file just before the closing body tag, or putting emergence.init() in a DOMContentLoaded may fix the solution, but I've fixed this in release 0.09
